MdeModulePkg/Core/Dxe/Gcd/Gcd.c

```diff
@@ -691,6 +697,14 @@ ConverToCpuArchAttributes (
     CpuArchAttributes |= EFI_MEMORY_WP;
   }
 
+  // MU_CHANGE BEGIN: Add EFI_MEMORY_SP
+
+  if ((Attributes & EFI_MEMORY_SP) == EFI_MEMORY_SP) {

```

Comment:
Maybe I missed it on the first pass, but is this bit supposed to be coexisting with the above attributes? Otherwise, should we make this part of the `else if` conditions?

Reply:
It can be in addition to the cacheability attributes.

Reply:
Added, as suggested.